Helper/Cart.php

```diff
@@ -3087,4 +3115,50 @@ public function setCurrentCurrencyCode($code)
             $this->store->setCurrentCurrencyCode($code);
         }
     }
+
+    /**
+     * Returns exists or generate new quote masked id
+     *
+     * @param int $quoteId
+     * @return string
+     * @throws \Exception
+     */
+    public function getQuoteMaskedId(int $quoteId): string
+    {
+        $quoteMaskedId = '';
+        try {
+            // we have different implementations of quoteIdToMaskedQuoteId in diff magento versions
```

Comment:
If it's not too hard please add the specific versions

Reply:
Removed `QuoteIdToMaskedQuoteIdInterface` it is totally missed on magento244, refactored to use just resource model.